### DISCUSSION TITLE
All Perms checkbox to New User addition

### DISCUSSION THREAD

**Discussion by RebornTrack970** [pelican-dev/panel#520] [Ideas]
Simply a small checkbox at the top of adding New User in a server's tabs that when checked adds all perms without the need to click each one.

**Comment by rmartinoscar** [comment]
Although it would be nice i wouldn’t check every perm with a single input, subusers shouldn’t be able to create late alone edit other subusers

**Comment by notAreYouScared** [comment]
https://github.com/pelican-dev/panel/commit/e6e3ce0f9d1665b9c6ee188f21d1f6310b341023 Closes this.
